### PR TITLE
Add runbook for KubevirtHyperconvergedClusterOperatorCRModification

### PR DESCRIPTION
**Pull request by erkanerol**
Signed-off-by: Erkan Erol <eerol@redhat.com>

### CODE REVIEW

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Impact
+
+When the operands are changed by someone else constantly, it may lead to oscillation in the cluster configuration and instabilities.
+
+
+## Diagnosis
+
+Check the alert details. `component_name` contains the type and name of operand that is overwritten.
```
Comment (sradco) [inline runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md:17]: `component_name` contains the type and name of the operand that is overwritten.

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way, and it overwrites its operands when there is an unexpected change on them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
```
Comment (sradco) [inline runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md:5]: Hyperconverged Cluster Operator configures clusters in an opinionated **way and overwrites** its operands when there is an unexpected change **to** them.

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md (hunk 1 above)
Comment (erkanerol) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: done

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md (hunk 2 above)
Comment (erkanerol) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: done

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way and overwrites its operands when there is an unexpected change to them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
```
Comment (orenc1) [inline runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md:5]: - I'm not sure if the term _configures clusters_ is accurate. Maybe _configures kubevirt and its supporting operators_ ?
- the HyperConverged object --> the HyperConverged *custom resource*

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+This alert fires when an operand of Hyperconverged Cluster Operator is changed by someone else.
```
Comment (orenc1) [inline runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md:7]: this alert *is fired*

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Check the alert details. `component_name` contains the type and name of the operand that is overwritten.
+
+```
+Labels
+    alertname=KubevirtHyperconvergedClusterOperatorCRModification
+    component_name=kubevirt-kubevirt-hyperconverged
+    severity=warning
+```
+In the example above, `kubevirt/kubevirt-hyperconverged` object is the operand for which the alert is triggered.
```
Comment (orenc1) [inline runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md:25]: to be more accurate we can say that the resource name itself contains the type, due to the fact that all kubevirt-hyperconverged CRs follow this pattern in their names: `<type>-kubevirt-hyperconverged`.

thus, if we use resource_type/resource_name notation, the example is:
`kubevirt.kubevirt.io/kubevirt-kubevirt-hyperconverged`

**Review by orenc1 — COMMENTED** [review]
in general looks fine. comments inline.

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md (hunk 3 above)
Comment (erkanerol) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: done

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md (hunk 4 above)
Comment (erkanerol) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: done

File: runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md (hunk 5 above)
Comment (erkanerol) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: @orenc1  Just updated. Could you please check it?

Comment (orenc1) [reply runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md]: yes, thanks.
lgtm on my side

**Pull request merged**